pypeit/spectrographs/magellan_fire.py

```diff
@@ -170,121 +213,160 @@ def check_frame_type(self, ftype, fitstbl, exprng=None):
         msgs.warn('Cannot determine if frames are of type {0}.'.format(ftype))
         return np.zeros(len(fitstbl), dtype=bool)
 
-    def bpm(self, shape=None, filename=None, det=None, **null_kwargs):
-        """
-        Override parent bpm function with BPM specific to X-Shooter VIS.
-
-        .. todo::
-            Allow for binning changes.
-
-        Parameters
-        ----------
-        det : int, REQUIRED
-        **null_kwargs:
-            Captured and never used
-
-        Returns
-        -------
-        bpix : ndarray
-          0 = ok; 1 = Mask
+    @property
+    def norders(self):
+        return 21
 
-        """
-        msgs.info("Custom bad pixel mask for FIRE")
-        self.empty_bpm(shape=shape, filename=filename, det=det)
-        if det == 1:
-            self.bpm_img[:, :4] = 1.
+    @property
```

Comment:
Add a TODO here that we somehow need to automate this, or make it a separate spectrograph.

Reply:
add a TODO here